modules/ROOT/pages/trusted-salesforce-org.adoc

```diff
@@ -1,6 +1,6 @@
 = Trusted Salesforce Organizations
 
-Enable cross-platform features between your Anypoint Platform organization and a trusted Salesforce organization (tenant) by establishing a connection between them. 
+Connect your Anypoint Platform organization with one or more trusted Salesforce organizations (tenants) to enable powerful cross-platform features. Establishing this tenant relationship allows seamless integration, such as syncing APIs via API Catalog, publishing invocable actions from RPA or Composer, and enhancing developer tools with Einstein. Manage these connections to control which features are active between the platforms.
```

Comment:
Spell out RPA?
Is it okay to use words like "powerful" and "seamless?" I don't mind, but writer.ai usually raps my knuckles.